modules/openfast-library/src/FAST_Subs.f90

```diff
@@ -27,7 +27,7 @@ MODULE FAST_Subs
    USE VersionInfo
 
    IMPLICIT NONE
-   
+
```

Comment:
For future pull requests, it would be nice if we didn't remove all the trailing spaces from existing source files. It creates a lot of unnecessary changes that can cause merge conflicts.

Reply:
Agreed.  That snuck in during one of the later PRs and I didn't catch it in time.

Reply:
For future comparisons, there is a `hide whitespace` option in the file compare in a GitHub PR.  Unfortunately, it hides all whitespace changes, not just the trailing ones.

Reply:
Yeah, I can _view_ these changes in the PR without the white spaces fine. I've just had experience with merge conflicts getting worse due to one file removing white space while another changes something else on the line. It's not _hard_ to fix those conflicts, just potentially time consuming (some merge-conflict editors make this easier than others).

Reply:
I agree.  I've also found it can really make resolving merge conflicts much more time consuming.  I'll pay more attention to that going forward.